test/integration-test/src/tests/load.rs

```diff
@@ -25,11 +30,19 @@ fn long_name() {
         .try_into()
         .unwrap();
     name_prog.load().unwrap();
-    name_prog.attach("lo", XdpFlags::default()).unwrap();
+
+    // create veth interface pair
+    let veth_name="aya-test-veth1";
```

Comment:
Please run rustfmt, clippy and other linters specified in https://github.com/aya-rs/aya/blob/main/.github/workflows/lint.yml locally - I'm sure they will complain about lack of spaces around `=`. I will approve the workflow CI checks afterwards.